charts/datadog/values.yaml

```diff
@@ -1194,6 +1199,11 @@ agents:
   ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
   affinity: {}
 
+  # agents.topologySpreadConstraints -- Allow the DaemonSet to schedule using pod topology spreading
+  ## By default, no constraints are set, allowing cluster defaults to be used for scheduling
+  ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+  topologySpreadConstraints: []
```

Comment:
Is there any real use case for the DaemonSet to use `topologySpreadConstraints`?

Reply:
Honestly, I can't conceive of a use case for it, but the spec says the field is applicable to DaemonSets, so I included it per the initial note in the PR. I can take it back out if you'd prefer too. Not sure what lint-docs wants, but I'll look into that too.

Reply:
You need to run `.github/helm-docs.sh` after bumping the chart version.
I think it's indeed better to remove the option, we try to avoid exposing options that could produce unexpected behaviour or could be misleading.